compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -207,12 +200,26 @@ static LogicalResult setRootConfigForPadPackPipeline(
   auto config = PackingConfigAttr::get(context, packingConfigLevels);
   setPackingConfig(linalgOp, config);
 
-  // Finish rest of tiling
+  // Do tiling
   // ------------------------------------------------------
   // -------------- Set lowering config -------------------
   // ------------------------------------------------------
-  auto tileK1 = findLargestFactor((int)tileK0 / (int)packedSizes[2],
+  // TODO (nmeshram) : We should be able to use fixed tiling config after we
+  // have padding support.
+  auto tileM1 = findLargestFactor((int)initShape[0], 16 * tilingScaleFactor,
+                                  (int)packedSizes[0]);
+  auto tileN1 = findLargestFactor((int)initShape[1], 16 * tilingScaleFactor,
+                                  (int)packedSizes[1]);
+  auto tileK1 = findLargestFactor((int)lhsShape[1] / (int)packedSizes[2],
                                   2 * tilingScaleFactor);
+
+  auto tileM0 =
+      findLargestFactor((int)initShape[0], 64 * tilingScaleFactor, (int)tileM1);
+  auto tileN0 =
+      findLargestFactor((int)initShape[1], 64 * tilingScaleFactor, (int)tileN1);
+  auto tileK0 = findLargestFactor((int)lhsShape[1], 256,
```

Comment:
I understand this is coming off late but any changes to KernelDispatch's logic should be supplemented with a lit test.

[compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy.mlir](https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy.mlir) is where the lit test should go - helps in seeing the immediate effect of new additions/updates to the existing tests.

Can we please address this in a follow up PR? @nirvedhmeshram

Reply:
done https://github.com/nod-ai/iree-amd-aie/pull/299